### ISSUE TITLE
Is there a way to restrict the areas of a page that are read?

### ISSUE THREAD

**Issue by tpanza** [VikParuchuri/marker#193]
I am having to process a large PDF document. It has some logos, boilerplate text, and other useless text in the top, bottom, left, and right margins of every page.

The model(s) seem to be struggling with recognizing these and turning them into Markdown text. It results in random gobbledygook at every PDF page boundary in the resulting Markdown file.

Might there be a way to pass in some settings so that these margin areas are ignored? I took at look in the settings.py (https://github.com/VikParuchuri/marker/blob/master/marker/settings.py) but didn't see anything about that.

I see on the README, "Removes headers/footers/other artifacts", but how do I control/tweak that?

**Comment by luc42ei** [comment]
yep, would be great to have that. in my case, I'd like to expand the area such that certain headers/footers are actually included because now they are omitted while containing important headings (e.g. of tables)

**Comment by luc42ei** [comment]
actually, one can expand the area by changing the BAD_SPAN_TYPES parameter in the settings.py file. it seems like removing all elements there would imply expanding the area to 100%

**Comment by svenha** [comment]
@luc42ei In version 1, there is no `BAD_SPAN_TYPES` anymore. What are you using now?

**Comment by Nevermetyou65** [comment]
Coming here for the sam thought. I see "Removes headers/footers/other artifacts" on the readme, but can not find a way to control it.

**Comment by gingemonster** [comment]
> [@VikParuchuri](https://github.com/VikParuchuri) How do I return headers and footers back into the text?

same question here
